stdio/file.c

```diff
@@ -666,6 +672,21 @@ off_t ftello(FILE *stream)
 }
 
 
+int fgetpos(FILE *stream, fpos_t *pos)
+{
+	off_t ret;
+	if ((stream == NULL) || (pos == NULL)) {
+		return set_errno(EINVAL);
+	}
```

Comment:
@agkaminski , perhaps these checks are redundant (not necessary). For example, on host, `getpos(f, NULL)` causes a segfault, so if you agree I'll remove them.

Reply:
Yeah, there's no need for NULL checks